src/compositionRoot.ts

```diff
@@ -0,0 +1,15 @@
+import { D2Api } from "./types/d2-api";
+import { Dhis2DataValueRepository } from "./data/Dhis2DataValueRepository";
+import { GetDataValuesUseCase } from "./domain/usecases/GetDataValuesUseCase";
+
+export function getCompositionRoot(api: D2Api) {
+    const dataValueRepository = new Dhis2DataValueRepository(api);
+
+    return {
+        dataValues: {
+            get: new GetDataValuesUseCase(dataValueRepository),
+        },
```

Comment:
As opposed to our general class approach on composition root, we're not memorizing the Use-case here, nothing to worry about for skeleton though.

Reply:
Do we need memoizing? It's an static object created when `getCompositionRoot` is called (at app start-up)

Reply:
That's right, nvm